src/components/Nav/Nav.js

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
+import Category from './Category/Category';
+import Dropdown from './Dropdown/Dropdown';
+import Sideoption from './Sideoption/Sideoption';
+import CATEGORY_DATA from './categoryData';
+import './Nav.scss';
 
 const Nav = () => {
-  return <h1>Nav</h1>;
+  // 상수데이터 => 서버데이터로 변경 예정
+  const categoryMap = CATEGORY_DATA.map(category => {
+    return (
+      <Category
+        key={category.id}
+        name={category.name}
+        carousel={category.carousel}
+      />
+    );
+  });
+
+  return (
+    <div className="navBar">
+      <img className="logo" alt="이미지로고" />
+      <section className="category">
+        <ul className="navList">{categoryMap}</ul>
```

Comment:
```suggestion
<ul className="navList">{CATEGORY_DATA.map(category => {
    return (
      <Category
        key={category.id}
        name={category.name}
        carousel={category.carousel}
      />
    );
  })}
  </ul>
```
방식에 차이이지만 개인적으로는 따로 분리하지 않고 바로 작성하는 게 더 직관적이라는 생각이 듭니다.